bin/si-conduit/src/command/remote/schema/push.ts

```diff
@@ -1395,10 +1393,26 @@ async function wrapInChangeSet(
         `Error creating schemas: ${unknownValueToErrorMessage(error)}`,
       );
     }
-    logger.info("Deleting changeset...");
+    logger.info("Deleting change set...");
     api.abandonChangeSet({
       workspaceId,
       changeSetId,
     });
   }
 }
+
+// Read input from the first keyboard press. We need to make sure we are no longer in raw mode
+// after we capture the input.
+async function readRawInput() {
+  Deno.stdin.setRaw(true);
+  let input: string | undefined;
+  try {
+    const buf = new Uint8Array(1);
+    await Deno.stdin.read(buf);
+    input = new TextDecoder().decode(buf).toLowerCase();
+    console.log(input);
+  } finally {
+    Deno.stdin.setRaw(false);
+    return input;
+  }
+}
```

Comment:
Good for now, but soon we'll want to migrate to use https://cliffy.io/docs@v1.0.0-rc.8/prompt/types/confirm out of `@cliffy/prompt/confirm`

Reply:
Ah good to call out!